packages/calcite-components/src/components/filter/filter.scss

```diff
@@ -1,3 +1,25 @@
+/**
+ * CSS Custom Properties
+ *
+ * @prop --calcite-filter-content-space: Specifies the padding of the component's content.
+ *
+ * @prop --calcite-filter-input-background-color: Specifies the input's background color.
+ * @prop --calcite-filter-input-border-color: Specifies the input's border color.
+ * @prop --calcite-filter-input-corner-radius: Specifies the input's corner radius.
+ * @prop --calcite-filter-input-shadow: Specifies the shadow around the input.
+ *
+ * @prop --calcite-filter-input-icon-color: Specifies the input's icon color.
+ * @prop --calcite-filter-input-text-color: Specifies the input's text color.
+ * @prop --calcite-filter-input-placeholder-text-color: Specifies the input's placeholder text color.
+ *
+ * @prop --calcite-filter-input-actions-background-color: Specifies the background color of the input's `clearable` and `number-button-type` elements.
+ * @prop --calcite-filter-input-actions-background-color-hover: Specifies the background color of the input's `clearable` and `number-button-type` elements when hovered.
+ * @prop --calcite-filter-input-actions-background-color-press: Specifies the background color of the input's `clearable` and `number-button-type` elements when pressed.
+ * @prop --calcite-filter-input-actions-icon-color: Specifies the icon color of the input's `clearable` and `number-button-type` elements.
```

Comment:
Nitpick: I don't think the "`number-button-type` elements" is relevant for the filter's use of input.